pkg/operator/ceph/nfs/config.go

```diff
@@ -137,7 +137,18 @@ RGW {
 }
 
 %url	` + url + `
+
+`
+
+	if version.IsAtLeast(cephver.Tentacle) {
```

Comment:
Do we know if tentacle is updated to ganesha 6? Anyway, let's discuss next week how to apply this setting in new vs upgraded clusters.

Reply:
We may want to try to sync with Ceph and Ganesha folks also to try to agree upon a Ceph upstream image version for the NFS v6 switchover.

IMO, making the switch from v5 to v6 would make sense for Squid -> Tentacle, and we could try to push for that as part of Ceph upstream's support/planning.

Reply:
I would vote that we don't even need this PR. When upgrading to ganesha 6, we can just put in our release notes that they need to restart the applications.

Reply:
Then we need this https://github.com/rook/rook/pull/15883 with v6

Reply:
I thought the change to numeric ID wasn't needed anymore. Since we can keep the alphanumeric id with v6, what changes are needed?

Reply:
3. Do nothing, and upgrades require application restart

This third option is what I was thinking, let's discuss...

Reply:
@parth-gr Did we reach a conclusion on this that no action is needed, and we can close this PR?